load_env.py

```diff
@@ -66,7 +68,29 @@ def load_env_file(run_type: str, file_string: Optional[str]=None) -> None:
         for key, value in env_dict.items():
             os.environ[key] = value
 
+    if Path(LOCAL_AWS_FILE).exists():
+        aws_config = _load_yaml_config(LOCAL_AWS_FILE)
+        config.update(aws_config)
+
+    _set_env_vars(config)        
+
     load_secrets()
+
+def _set_env_vars(config: dict) -> None:
+    for key, value in config.items():
+        if key not in os.environ:
+            os.environ[key] = str(value)    
+
+def _load_yaml_config(file_path: str) -> None:
+    try:
+        with open(file_path, "r") as file:
+            return yaml.safe_load(file) or {}
+    except FileNotFoundError:
+        print(f"Warning: Config file {file_path} not found.")
+        return {}
+    except yaml.YAMLError as err:
+        print(f"Error parsing YAML file {file_path}: {err}")
+        return {}            
```

Comment:
Let's just do 1 except block with no print statements as for the most part this file won't exist in other environments. 

```
except:
  return {}
```